apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx

```diff
@@ -139,7 +139,7 @@ export class LibraryManagerDialog extends React.Component {
         this.setState({classLibraries, projectLibraries});
       },
       error => {
-        console.log('error: ' + error);
+        this.setState({classLibraries: null});
```

Comment:
This feels a little fragile. i.e. it looks like we're using `classLibraries === null` as a flag for an error state. It's possible that a downstream api change could end up with a valid state where `classLibraries` is set to `null`.
Possible suggestion: Reuse `this.state.error` - It looks like it gets used for a similar purpose and could potentially be reused.

Reply:
+1 -- i think `this.state.error` will need an additional "type" though. right now, `this.state.error` is always displayed at the bottom of the dialog after all content:

https://github.com/code-dot-org/code-dot-org/blob/3ab7b824f6650d3f472d2125373092d2ac35f995/apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx#L409

Reply:
It's possible that both errors could happen at the same time, right? Like, we're unable to load class libraries, but then you try to import from an id and that also fails - in that case we would want to show both error messages I think, right?

Reply:
yeah, that's a good point. so rather than both errors using `this.state.error`, we could refactor `this.state.error -> this.state.importError` and the new state can be `this.state.classLibraryError`?